pom.xml

```diff
@@ -283,14 +251,14 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-dependency-plugin</artifactId>
         <configuration>
+          <!-- The items listed below have been verified required to override. -->
           <ignoredUnusedDeclaredDependencies>
-            <!-- ignore transitive dependency overridden to avoid vulnerability -->
+            <ignored>cglib:cglib</ignored>
```

Comment:
What is cglib doing here? Why must it be ignored? Why is it added at all?

Reply:
cglib is already a transient dependency from commons digester but its not on any remotely modern copy.  So we are excluding the bad copy in favor of the last one released.

Reply:
Oh, I see. It's because you added cglib to the dependencies instead of the dependencyManagement section. I think it would need to be in the dependencyManagement section to manage the version for the resolver. It doesn't need to be ignored... since it never needs to be specified as a direct dependency.